src/gui/Src/Gui/CPUInfoBox.cpp

```diff
@@ -244,6 +267,52 @@ void CPUInfoBox::disasmSelectionChanged(dsint parVA)
     if(getInfoLine(0) == getInfoLine(1)) //check for duplicate info line
         setInfoLine(1, "");
 
+    // check references details
+    // code extracted from ExtraInfo plugin by torusxxx
+    XREF_INFO xrefInfo;
+    DbgXrefGet(parVA, &xrefInfo);

```

Comment:
The usage of `DbgXrefGet` is incorrect and will cause memory leak. The ExtraInfo plugin has been updated to fix it. Please use the newer version of ExtraInfo.